Control.cpp

```diff
@@ -7,14 +7,19 @@
 */
 #include "Control.h"
 
-Control::Control() {}
+Control::Control(){}
+
+Control::Control(BNO *c, SensorMap *s) {
```

Comment:
BNO* bno, SensorMap* map

Have a reasonable name for them. Remember to change it too on the .h